src/main/java/com/example/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.TechItEasy.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
```

Comment:
Een @RequestMapping("/televisions") is hier misschien handig, omdat je elke mapping met /televisions begint